kili/mutations/project/__init__.py

```
@@ -187,6 +189,8 @@ def create_project(self, input_type: str, json_interface: dict,
             For more detailed examples on how to create projects,
                 see [the recipe](https://github.com/kili-technology/kili-python-sdk/blob/master/recipes/create_project.ipynb).
         """
+        if input_type == 'FRAME':
+            warnings.warn("FRAME input type is deprecated. Please use VIDEO instead")
```

same here, we should rather raise an error